roles/ceph-osd/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: install dependencies
+  apt: pkg=parted
+       state=present
+
+- name: write out bootstrap keyring
+  copy:
+    dest: "{{ hostvars[groups['mons'][0]]['keyrings']['source'] }}"
+    content: "{{ hostvars[groups['mons'][0]]['keyrings']['content'] | b64decode }}"
+
+- name: prepare osd disk(s)
```

Comment:
Rerunning this task would result in:
```
ok: [159.8.186.12] => (item=['/dev/sdg', '/dev/sdc']) => {"changed": false, "cmd": ["ceph-disk", "prepare", "/dev/sdg", "/dev/sdc"], "delta": "0:00:00.056950", "end": "2015-08-13 16:26:36.363460", "failed": false, "failed_when_result": false, "item": ["/dev/sdg", "/dev/sdc"], "rc": 1, "start": "2015-08-13 16:26:36.306510", "stderr": "ceph-disk: Error: Device is mounted: /dev/sdg1", "stdout": "", "warnings": []}
```
which prevents it from being re-prepared (which is what we wanted).